app/helpers/error_items_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ErrorItemsHelper
+  def error_items(field)
+    if flash[:validation] && flash[:validation].select { |key| key.to_s.match(field) }.any?
+      sanitize(flash[:validation]
+        .select { |key| key.to_s.match(field) }
+        .map { |error| error[:text] }
+        .join("<br>"))
```

Comment:
This markup seems a bit odd - is this rendering a "list" of errors as text separated by `<br>`s

@alex-ju  - is that acceptable do you think?

Reply:
The [error message components support arrays](https://components.publishing.service.gov.uk/component-guide/error_message/with_items) ideally, we'll get the served as such.